### ISSUE TITLE
macos build Godot v.3.2 from source fails

### ISSUE THREAD

**Issue by salvob41** [godotengine/godot#43090]
When building for macOSX from source it shows the following error:

```
$ scons platform=osx arch=x86_64 --jobs=$(sysctl -n hw.logicalcpu)
scons: Reading SConscript files ...
xcrun: error: SDK "iphoneos" cannot be located
xcrun: error: SDK "iphoneos" cannot be located
xcrun: error: unable to lookup item 'SDKVersion' in SDK 'iphoneos'
Failed to find SDK version while running xcrun --sdk iphoneos --show-sdk-version.
CalledProcessError: Command '['xcrun', '--sdk', 'iphoneos', '--show-sdk-version']' returned non-zero exit status 1.:
  File "/Users/panoramix/Desktop/coding/notapixelstudio/godot/SConstruct", line 43:
    if detect.can_build():
  File "./platform/iphone/detect.py", line 17:
    if get_darwin_sdk_version("iphone") < 13.0:
  File "/Users/panoramix/Desktop/coding/notapixelstudio/godot/methods.py", line 717:
    return float(decode_utf8(subprocess.check_output(["xcrun", "--sdk", sdk_name, "--show-sdk-version"]).strip()))
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/subprocess.py", line 411:
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/subprocess.py", line 512:
    raise CalledProcessError(retcode, process.args,
```

**Godot version:**
Godot v.3.2 - commit 4ece7c287f7e3cb88968604e6b0fa39b274954f0

**OS/device including version:**
macOS 10.15.7

**Comment by bruvzg** [comment]
What Xcode version do you have installed, or is it only "Xcode Command Line Tools" ?

**Comment by salvob41** [comment]
@bruvzg it's only command line tools. 

I should add that it works with `master` branch. And it worked around a week ago also on godot3.2

**Comment by akien-mga** [comment]
Fixed by #43091.
